pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -967,10 +967,10 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 				podExists = true
 			}
 
-			if network.NetworkSource.Multus != nil {
+			if multusOrNpwgNetworkName := util.GetMultusOrNpwgNetworkName(network); multusOrNpwgNetworkName != "" {
```

Comment:
this logic isn't exactly the same as before. 

previously we were checking if the multus object existed, now we're checking if either the multus/npwg object exists and whether or not the name is set as well.

I think this means the 'networkNameExistsOrNotNeeded' won't get set the same way it was previously.

Reply:
done